duckduckgo/duckduckgo-autofill
# Improve narrow width iframes

**Reviewer:** @alistairjcbrown 
**Asana:** https://app.asana.com/0/0/1204100616987020/f

## Description
On extensions, our tooltip could be cut off in iframes. This attempts to mitigate the issue by centering the tooltip when the left border is out of the visible frame and when it's overflowing from the right side it just moves it slightly inwards.

## Steps to test
Automated tests added. To see it in action you can check the iframe signup form at reddit.com and https://gravida.pro/nytimes/New%20York%20Times%20Checkout (note that I uploaded the form to one of my sites because nytimes is pretty aggressive with bot blocking and I wasn't able to test after just a couple of refreshes).
